src/main/java/com/renatusnetwork/momentum/data/cmdsigns/CmdSignsDB.java

```diff
@@ -76,10 +86,31 @@ public static void unuseCommandSign(String uuid, String name) {
         );
     }
 
-    public static void updateCommand(String name, String newCommand) {
+    public static void updateCommand(String name, String oldCommand, String newCommand) {
```

Comment:
Is there actually a case where this can happen?

Reply:
the command takes the index of the sign's command rather than the player needing to type out the whole sign's command, and it gets translated into the actual string for db purposes. if you mean the `LIMIT 1` condition, there's no check on whether a sign has duplicate commands, so if a sign has 2 commands that are identical they'd both be changed otherwise unless we want to disallow duplicate commands on a single sign.

Reply:
The LIMIT 1 will be useless though as primary keys cannot be repeated. So if someone tries to add another command of the same command to a sign, it will error out on the db since the priamry key is (name, command) and those will be the same. We should add a check on the modify and add commands so that it prevents duplicate commands